plugins/generator-1.14.4/forge-1.14.4/procedures/itemstack_remove_specific_enchantment.java.ftl

```diff
@@ -0,0 +1,5 @@
+Map<Enchantment, Integer> enchantments = EnchantmentHelper.getEnchantments(${input$item});
```

Comment:
Wrap whole procedure with {}. Otherwise, the scope of the enchantments variable is whole procedure and the second time this block will be used, variable will be duplicated and build will fail. Also rename enchantments to _enchantments to avoid conflicts with names of local variables (_ is standard for built-in variables to avoid conflicts).

Do same for other MC version and other cases of this.

Reply:
It has been fixed!